container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/user_services_functions/start_user_services.go

```diff
@@ -532,16 +536,37 @@ func getUserServicePodContainerSpecs(
 
 	resourceLimitsList := apiv1.ResourceList{}
 	resourceRequestsList := apiv1.ResourceList{}
+
+	err = checkIfResourcesAreSetProperly(minCpuAllocationMilliCpus, cpuAllocationMillicpus, apiv1.ResourceCPU)
```

Comment:
The K8S API validates those already: https://github.com/kubernetes/kubernetes/blob/5a5ebfd88b2899f2116dc858100832f30ac33cf3/pkg/apis/core/v1/validation/validation.go#L38